solutions/fully-configurable/variables.tf

```diff
@@ -0,0 +1,356 @@
+  }
+}
+
+variable "access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the resources created by this solution. For more information, see https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial."
+  default     = []
+}
+
+
+########################################################################################################################
+# Event Notifications
+########################################################################################################################
+
+variable "service_credential_names" {
+  type        = map(string)
+  description = "The mapping of names and roles for service credentials that you want to create for the Event Notifications instance. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/tree/main/solutions/standard/DA-types.md#service-credential-secrets"
+  default     = {}
+
+  validation {
+    condition     = alltrue([for name, role in var.service_credential_names : contains(["Manager", "Writer", "Reader", "Event Source Manager", "Channel Editor", "Event Notification Publisher", "Status Reporter", "Device Manager", "Email Sender", "Custom Email Status Reporter"], role)])
+    error_message = "The specified service credential role is not valid. The following values are valid for service credential roles: 'Manager', 'Writer', 'Reader', 'Event Source Manager', 'Channel Editor', 'Event Notification Publisher', 'Status Reporter', 'Device Manager', 'Email Sender', 'Custom Email Status Reporter'"
+  }
+}
+
+variable "event_notifications_name" {
+  type        = string
+  description = "The name of the Event Notifications instance that is created by this solution. If a `prefix` input variable is specified, it is added to this name in the `<prefix>-value` format."
+  default     = "base-event-notifications"
+}
+
+variable "service_plan" {
+  type        = string
+  description = "The pricing plan of the Event Notifications instance. Possible values: `Lite`, `Standard`"
+  default     = "standard"
+  validation {
+    condition     = contains(["lite", "standard"], var.service_plan)
+    error_message = "The specified pricing plan is not available. The following plans are supported: `Lite`, `Standard`"
+  }
+}
+
+variable "service_endpoints" {
+  type        = string
+  description = "Specify whether you want to enable public, or both public and private service endpoints. Possible values: `public`, `public-and-private`"
+  default     = "public-and-private"
+  validation {
+    condition     = contains(["public", "public-and-private"], var.service_endpoints)
+    error_message = "The specified service endpoint is not supported. The following endpoint options are supported: `public`, `public-and-private`"
+  }
+}
+
+variable "event_notifications_tags" {
```

Comment:
`event_notifications_resource_tags` (to difference it with access tags which actually seems to be missing from the module. Created a tracking [issue](https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/issues/400) for that)